discovery-cloud-secure.md

```diff
@@ -0,0 +1,31 @@
+
+# Discover using the cloud, but otherwise still secure
+## QR + plex + LE
+  * (box will temporarily set up the bridge and update DNS during LE reg/renew)
+  * con: you need to trust not only the Box and the client, but also the bridge/DNS-server
+  * pro: no need for Cordova; just use a standard QR-code reader and mobile browser
+  * con: scanning a QR code is cumbersome on laptops and TVs
+
+## Flow:
+1. At build time, a serial number and a deploy token is put onto the box, so that it can request two services:
+ * setting up a DNS entry,
+ * signing a CSR.
```

Comment:
I'm sure you just assume this, but as part of this process the box needs to generate an RSA keypair (unique, and never leaves the device)